runtime/src/iree/tooling/trace_replay.c

```diff
@@ -543,14 +552,77 @@ static void iree_trace_replay_write_element(
 }
 
 // Simple deterministic pseudorandom generator.
-// Typically in tests we want reproducible results both across runs and across
-// machines.
+// This function is same as C++'s std::minstd_rand.
+static uint32_t iree_tree_replay_pseudorandom_uint32(uint32_t* state) {
+  *state = (*state * 48271) % RAND_MAX;
```

Comment:
No, don't use RAND_MAX here! It could be anything. We care about the specific value that we were using here before. If it becomes a different value, then the above comment `// This function is same as C++'s std::minstd_rand.` becomes incorrect.  What I was saying was , give a name to *that* value.